llvm/test/Transforms/MemCpyOpt/memcpy-byval-forwarding-clobbers.ll

```diff
@@ -16,14 +16,10 @@ declare void @llvm.memcpy.p0.p0.i64(ptr noalias nocapture writeonly, ptr noalias
 define i1 @alloca_forwarding_lifetime_end_clobber() {
```

Comment:
Should update the comment above here, because we now do the optimization by dropping the lifetime intrinsics.